### DISCUSSION TITLE
EU funds guidance typo

### DISCUSSION THREAD

**Discussion by odscjen** [OP-TED/eForms-SDK#184] [Q&A] [answered]
In https://docs.ted.europa.eu/eforms/1.1.1/schema/all-in-one.html#EUFundsSection it states that "EU Funds Details (BT-6140) may only be provided once for each BG-611." But in https://docs.ted.europa.eu/eforms/1.2.1/reference/business-terms/index.html#BG-611 BT-6140 isn't part of BG-611. I'm guessing this is a typo but should it be "Contract EU Funds Details (BT-6110) may only be provided once for each BG-611" OR "EU Funds Details (BT-6140) may only be provided once for each BG-614"

**Comment by YvesJo** [comment, marked as answer]
Thanks for pointing out this typo. It should indeed read: "EU Funds Details (BT-6140) may only be provided once for each BG-61**4**"
Documentation will be corrected accordingly.
